bin/pycbc_multi_inspiral

```diff
@@ -93,6 +93,28 @@ def calculate_antenna_pattern(args, sky_pos_indices):
             )
     return antenna_pattern
 
+def slide_limiter(args):
+    '''
+    This function computes the number of shortslides used by the coherent
+    matched filter statistic to obtain as most background triggers as 
+    possible.
+
+    It it bounds the amount of slides to avoid counting triggers more 
+    than once.
```

Comment:
```suggestion
    It bounds the number of slides to avoid counting triggers more than once.
    If the data is not time slid, there is a single slide for the zero-lag.
```